conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -106,6 +110,7 @@ private TypeSpec generate(
                         isInStagedBuilderMode(builderInterfaceClass) ? "DefaultBuilder" : "Builder")
                 .addAnnotation(ConjureAnnotations.getConjureGeneratedAnnotation(BeanBuilderGenerator.class))
                 .addModifiers(Modifier.STATIC, Modifier.FINAL)
+                .addField(FieldSpec.builder(boolean.class, BUILT_FIELD).build())
```

Comment:
How much more expensive would a volatile be here?

Reply:
Not particularly compared to other operations, but nothing in the builder is threadsafe so I don't think it would make much sense.

Reply:
Yeah, I think you're right. Although it would probably detect some badness if someone *WERE* to do something really crazy like pass a builder around...